lib/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueFactory.kt

```diff
@@ -118,4 +123,13 @@ class HmppsQueueFactory(
           )
         )
       }
+
+  fun createJmsListenerContainerFactory(awsSqsClient: AmazonSQS): DefaultJmsListenerContainerFactory =
+    DefaultJmsListenerContainerFactory().apply {
+      setConnectionFactory(SQSConnectionFactory(ProviderConfiguration(), awsSqsClient))
+      setDestinationResolver(DynamicDestinationResolver())
+      setConcurrency("1-1")
+      setSessionAcknowledgeMode(Session.CLIENT_ACKNOWLEDGE)
+      setErrorHandler { t: Throwable? -> log.error("Error caught in jms listener", t) }
+    }
```

Comment:
very minor, but if you got two listeners it would be nice to know which of listeners failed - e.g log queue name. I suspect we have lost that context by now though

Reply:
Yeah it's a good point, I think we can manage that.